### PR TITLE
Only load wallet if wallet has not already been loaded

### LINKED ISSUE #3327: Wallets can be loaded twice in WalletManager
Calling `WalletManager.Load` before `WalletManager.Start` will result in the same wallet being loaded twice.  This causes issues, particularly with `WalletManager.GetWalletByName`, which throws an exception if more than one wallet with the same name is loaded.

### PR DESCRIPTION
**Pull request by rowandh**
Closes #3327.

### CODE REVIEW

File: src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs [hunk 1]
```diff
@@ -1363,7 +1363,7 @@ private Wallet GenerateExtPubKeyOnlyWalletFile(string name, DateTimeOffset? crea
         }
 
         /// <summary>
-        /// Loads the wallet to be used by the manager.
+        /// Loads the wallet to be used by the manager if a wallet with this name has not already been loaded.
```
Comment (zeptin) [inline src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs]: Not related to this PR, but the trace log in this method is quite odd (`NOT_FOUND`). Maybe a typo?

**Comment by dangershony** [comment]
This should to do the release as well

**Comment by dangershony** [comment]
I wonder if `Load()` needs to go inside the lock?

**Comment by bokobza** [comment]
@rowandh can you please add this to release/3.0.2.0? I'm happy to do it otherwise.

**Comment by rowandh** [comment]
@bokobza yep, I will do it

**Comment by bokobza** [comment]
Excellent, thanks mate.

**Pull request merged**